src/modules/deal-folder/deal-folder.service.ts

```diff
@@ -95,23 +87,17 @@ export class DealFolderService {
     and driveId appears to necessary because using the ukefSiteId and drive name only appears to work when accessing drives, not items *within* drives. */
     const fileDestinationPath = `${buyerName}/D ${dealId}`;
 
-    return `${this.config.baseUrl}/sites/${sharepointSiteId}/drives/${driveId}/root:/${fileDestinationPath}/${fileName}:/createUploadSession`;
+    return `${this.sharepointConfig.baseUrl}/sites/${sharepointSiteId}/drives/${driveId}/root:/${fileDestinationPath}/${fileName}:/createUploadSession`;
```

Comment:
We could move this function (and others referencing upload urls) to sharepoint service -- my only issue with that is that it's breaking from the pattern,

Reply:
It's a tricky one to place, I'm happy to leave it where it is for now :)